app/src/main/java/net/squanchy/support/graphics/Color.kt

```diff
@@ -0,0 +1,29 @@
+package net.squanchy.support.graphics
+
+import android.graphics.Color
+import android.support.annotation.ColorInt
+
+private const val MAX_COLOR_COMPONENT = 255.0
+private const val GAMMA_FACTOR = 2.2
+private const val MAX_LIGHTNESS_FOR_LIGHT_TEXT = .18
+private const val FACTOR_RED = 0.2126
+private const val FACTOR_GREEN = 0.7151
+private const val FACTOR_BLUE = 0.0721
+
+@ColorInt
+internal fun Int.contrastingTextColor(darkTextColor: Int, lightTextColor: Int): Int {
```

Comment:
should the params be annotated with `@ColorInt`?

Reply:
Yes :) 89e3d2bbc56e2e6fffedbb9341481f74635d7554